### ISSUE TITLE
Date gets duplicated with each new activity

### ISSUE THREAD

**Issue by lluchs** [JacobEvelyn/friends#71]
```sh
-> % friends --version
friends version 0.10
-> % friends add friend Anton
Friend added: "Anton"
-> % friends add friend Berta
Friend added: "Berta"
-> % friends add friend Cäsar
Friend added: "Cäsar"
-> % friends add activity "Foo with Anton and Berta"
Activity added: "2016-01-12: Foo with Anton and Berta"
-> % cat friends.md
### Activities:
- 2016-01-12: Foo with **Anton** and **Berta**

### Friends:
- Anton
- Berta
- Cäsar
-> % friends add activity "2016-01-10: Bar with Anton and Cäsar"
Activity added: "2016-01-10: Bar with Anton and Cäsar"
-> % cat friends.md
### Activities:
- 2016-01-12:  2016-01-12: Foo with **Anton** and **Berta**
- 2016-01-10: Bar with **Anton** and **Cäsar**

### Friends:
- Anton
- Berta
- Cäsar
-> % friends add activity "2016-01-09: Baz with Anton, Berta and Cäsar"
Activity added: "2016-01-09: Baz with Anton, Berta and Cäsar"
-> % cat friends.md
### Activities:
- 2016-01-12:  2016-01-12:  2016-01-12: Foo with **Anton** and **Berta**
- 2016-01-12:  2016-01-10: Bar with **Anton** and **Cäsar**
- 2016-01-09: Baz with **Anton**, **Berta** and **Cäsar**

### Friends:
- Anton
- Berta
- Cäsar

```

(The example isn't as minimal as it could be, because I suspected it was related to the amount of friends you specify, but I hope it's clear enough.)

**Comment by JacobEvelyn** [comment]
Yikes! Thanks for spotting this; I'll look into it tonight.

**Comment by akleemans** [comment]
Experiencing the same behaviour (0.10):

    > friends add friend Anton
    Friend added: "Anton"
    > friends add activity "Skateboarding with Anton"
    Activity added: "2016-01-12: Skateboarding with Anton"
    > friends add friend Berta
    Friend added: "Berta"
    > friends list activities
    2016-01-12:  2016-01-12:  2016-01-12: Skateboarding with Anton

**Comment by JacobEvelyn** [comment]
Sorry for the delay! Looking into this now.

**Comment by JacobEvelyn** [comment]
I'm going to `yank` this version from RubyGems.

**Comment by JacobEvelyn** [comment]
Looks like this is a problem with the changing of the `Activity` deserialization regex. I've got a fix on its way.

**Comment by JacobEvelyn** [comment]
I'll bundle this into version 0.11 in a few minutes.

**Comment by JacobEvelyn** [comment]
This is released. Note that because of #72 you'll have to do a normal `gem update friends && gem cleanup friends` instead of just `friends update`. That issues has also been fixed in version 0.11.
